R/RunDiagnostics.R

```diff
@@ -110,6 +110,8 @@ getDefaultCovariateSettings <- function() {
 #'                                    diagnostics to.
 #' @param cohortDefinitionSet         Data.frame of cohorts must include columns cohortId, cohortName, json, sql
 #' @param cohortTableNames            Cohort Table names used by CohortGenerator package
+#' @param conceptCountsTable          Concepts count table name. The default is "#concept_counts" to create a temporal concept counts table.
```

Comment:
THese changes look good but you need to run `devtools::document()` (or the build in rstudio) to add the changes to  include these parameters.  Doing this should get the build to finish and I can then merge in your changes.